### PR TITLE
VACMS-4200: d9 compatibility for vagovadmin

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #4200 

## Testing done

Manual testing

## Screenshots

Screen Shot 2021-02-17 at 4 09 15 PM

## QA steps

1. Go to /coronavirus-veteran-frequently-asked-questions
- [ ] tooltips on content proofing are refactored to use a new pattern established as a part of #3984 and are displayed without errors
- [ ] `vagovadmin` theme is compatible with Drupal 9
   - [ ] `core/jquery.ui.tooltip` library dependency is removed

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by oksana-c — COMMENTED** [review]
we should include a refactoring form a `span` to a `button` element in this template - https://github.com/department-of-veterans-affairs/va.gov-cms/blob/master/docroot/themes/custom/vagovadmin/templates/layout/field--node--page.html.twig#L72

**Comment by acrollet** [comment]
> we should include a refactoring form a `span` to a `button` element in this template 

good call!

File: docroot/themes/custom/vagovadmin/assets/scss/_tooltip.scss [hunk 1]
```diff
@@ -4,7 +4,9 @@
  */
 
 .proofing-element-help {
+  -webkit-appearance: none;
```
Comment (oksana-c) [inline docroot/themes/custom/vagovadmin/assets/scss/_tooltip.scss:7]: this should likely be `appearance: none;` and the auto-prefixer (iirc we should have it setup) should take care of adding non-standard versions of this property.

**Review by oksana-c — CHANGES_REQUESTED** [review]
⚡ 👍 
left one change request re: css property

File: docroot/themes/custom/vagovadmin/assets/scss/_tooltip.scss (hunk 1 above)
Comment (acrollet) [reply docroot/themes/custom/vagovadmin/assets/scss/_tooltip.scss]: good call, thanks! I checked and you're right about the auto-prefixer, however I had added this at one point while fixing the button CSS but it wasn't actually necessary - removed entirely!

**Review by oksana-c — APPROVED** [review]

File: docroot/themes/custom/vagovadmin/templates/layout/field--node--page.html.twig [hunk 2]
```diff
@@ -69,9 +69,9 @@
     <div{{ title_attributes.addClass(title_classes) }}>
       {{ label }}
       {% if description %}
-        <span class="proofing-element-help" role="tooltip" tabindex="0" data-proofing-help-title="About '{{ label }}' field" data-proofing-help="{{ description|raw }}">
+        <button class="proofing-element-help" role="tooltip" tabindex="0" data-proofing-help-title="About '{{ label }}' field" data-proofing-help="{{ description|raw }}">
```
Comment (oksana-c) [inline docroot/themes/custom/vagovadmin/templates/layout/field--node--page.html.twig:72]:
```suggestion
        <button class="proofing-element-help" role="tooltip" data-proofing-help-title="About '{{ label }}' field" data-proofing-help="{{ description|raw }}">
```

**Review by oksana-c — APPROVED** [review]

**Pull request merged**